gemini/use-cases/retrieval-augmented-generation/code_rag.ipynb

```diff
@@ -0,0 +1,1231 @@
+{
```

Comment:
<p>Always say "Vertex AI" not just "Vertex"</p><p> </p><p>LangChain should always be written with this capitalization.</p><p> </p><p>Be consistent in naming for "Tree Sitter"</p><p> </p><p>Be consistent in capitalization for "repo". Also, it should be spelled out "repository"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/481/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Done</p>